torch/utils/hipify/cuda_to_hip_mappings.py

```diff
@@ -3845,6 +3845,7 @@
             ),
         ),
         ("cudaDataType_t", ("hipDataType_t", CONV_TYPE, API_RUNTIME, HIP_UNSUPPORTED)),
+        ("cudaDataType_t", ("hipDataType", CONV_TYPE, API_RUNTIME, HIP_UNSUPPORTED)),
```

Comment:
Which one does `cudaDataType_t` should be mapped to? I think we have to keep this line or the line above in the mapping table because only one takes effect right?.

Reply:
this is why this logic is here: https://github.com/ROCmSoftwarePlatform/pytorch/pull/1257/files/8eb336d2325eadd66e5bc402cd2d9586a12e7f95#diff-74fcb26047c1df4024105d36ce22a36b77cf8cc93c28631d743e639b3d6066aeR594